docs/List.md

```diff
@@ -0,0 +1,466 @@
+        <Statistic
+          title='Total Session'
+          value='25'
+        />
+      </Carousel.Tile>
+      <Carousel.Tile>
+        <Statistic
+          title='Total Hours'
+          value='211'
+        />
+      </Carousel.Tile>
+      <Carousel.Tile>
+        <Statistic
+          title='Total Cost (AUD)'
+          prefix='AUD'
+          value='54,118.00'
+        />
+      </Carousel.Tile>
+      <Carousel.Tile>
+        <Statistic
+          title='Total Cost (DZD)'
+          prefix='DZD'
+          value='99.00'
+        />
+      </Carousel.Tile>
+    </Carousel>
+</>
+```
+
+
+### Pagination section
+All list should have [Pagination](/#/Interaction/Pagination) when the page is customer facing
+It should be hidden when there is no record (also no result when filtering)
+
+```jsx
+import { PaginationDetails, PaginationButtons } from '@Domain/Paginators';
+import { GridLayout } from '@Domain/Layouts';
+import { Variables } from '@Common';
+  
+  initialState = { page: 1 };
+  
+  const props = { totalPages: 11, totalCount: 105, pageSize: 10};
+  
+  <GridLayout
+      gutterMarginY={Variables.Spacing.sXSmall}
+      horizontalAlignment={GridLayout.HorizontalAlignment.Justify}
+      verticalAlignment={GridLayout.VerticalAlignment.Middle}
+    cells={[
+            {
+               size: { desktop: 6, min: 12 },
+               content: <PaginationDetails
+                          currentPage={state.page}
+                          totalPages={props.totalPages}
+                          totalCount={props.totalCount}
+                          pageSize={props.pageSize}
+                        />,
+               displayType: 'flex',
+               flexHorizontalAlignment: { desktop: GridLayout.HorizontalAlignment.Left, min: GridLayout.HorizontalAlignment.Center }
+             },
+             {
+               size: { desktop: 6, min: 12 },
+               content: <PaginationButtons
+                          currentPage={state.page}
+                          totalPages={props.totalPages}
+                          handlePageChange={(page) => setState({page})}
+                        />,
+               displayType: 'flex',
+               flexHorizontalAlignment: { desktop: GridLayout.HorizontalAlignment.Right, min: GridLayout.HorizontalAlignment.Center }
+             }
+           ]}
+  />
+```
+
+### Loading state
+List section (from top to bottom): filter and sort section, statistics tile, cards, pagination with sLarge spacing 24px in between
+
+Filter and sort section goes together as a 40px height skeleton
+Statistics tile would depend on the number of tile would the list have (It would be depend on minimum number of tile so it would not include the optional ones)
+The skeleton would be 200 x 90 with 8 margin in between Cards section skeleton would have three cards only and the skeleton height would depend on the card height on that page
+
+Pagination section would not have a skeleton for it
```

Comment:
```When loading content we display skeletons (not spinners). The skeletons should reflect the structure of the content that will be displayed once the page has loaded. 

The following values will typically work for most content, but depending on the size of your content they may need minor adjustments.

Filter Section: 40px height
Statistics: 200px wide | 90px height | sLarge spacing between 
Cards: 78px height | sMedium spacing between

No skeleton for pagination bar

Only display the minimum number of Statistics/Cards that the page can load with. If it always has 6 Statistics, then show 6 skeletons. For lists of Cards, display as many as a full page of results (typically 10) unless the content is unlikely to reach a full page of results.
```

Reply:
should card skeleton depend on the content or fixed height?

Reply:
Card skeleton should match the content height of how big the cards are going to be. 78px is the normal size right?